roles/wildfly_systemd/tasks/main.yml

```diff
@@ -15,16 +15,26 @@
   when:
     - wildfly_enable_yml_config
     - eap_enable is defined and eap_enable
+    - eap_apply_cp is defined and eap_apply_cp
 
```

Comment:
This tasks also needs the eap 7 version check as in line 29 below (or it will fail on eap 8 when no patches are available)